ush/ICpert_scripts/add_soil_layers.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+print_info_msg "
+Interpolating soil variables from the 4 layers in NOAH LSM (in GFS output)
```

Comment:
While Noah LSM is in GFS, this work is specific to GEFS output, so it would be good to add it.

Reply:
Ok, I edited this for clarity.  Take a look at the new version once I push and let me know what you think.